sunpy/net/dataretriever/tests/test_goes_suvi.py

```diff
@@ -89,13 +89,13 @@ def test_fetch_working(suvi_client):
     """
     start = '2019/05/25 00:50'
     end = '2019/05/25 00:52'
+    # TODO: This search is super slow, is it a server issue or our code
```

Comment:
Can you open an issue for this?

Reply:
https://github.com/sunpy/sunpy/issues/4099